web-common/src/features/scheduled-reports/utils.ts

```diff
@@ -17,3 +67,93 @@ export function getDashboardNameFromReport(
     null
   );
 }
+
+export function extractNotification(
```

Comment:
I only see this function called once, it doesn't use the `userEmail` parameter, and it only uses `isEdit=true`, so we could remove those two parameters? Or we could consolidate and use `extractNotificationV2`.

Reply:
Moving this back to alerts. It is till using the old forms lib. We can merge them once we move it to superforms.